crates/miden-lib/src/transaction/inputs.rs

```diff
@@ -104,6 +104,7 @@ impl TransactionAdviceInputs {
     ///     number_of_input_notes,
     ///     TX_SCRIPT_ROOT,
     ///     TX_SCRIPT_ARGS_KEY,
+    ///     AUTH_PROCEDURE_ARG,
```

Comment:
nit: I would use plural "arguments" - i.e., `AUTH_PROCEDURE_ARGS` this would be consistent with "note args" and "transaction script args".